aether-utils/aether-client/aether/client_test/fixtures/mappings/mapping.json

```diff
@@ -0,0 +1,31 @@
+{
+	"entities": {
+
+	},
+	"mapping": [
+		[
+			"facility_id",
+			"CurrentStock.facility_id"
+		],
+		[
+			"#!uuid",
+			"CurrentStock.id"
+		],
+		[
+			"#!uuid",
+			"CurrentStock.rev"
+		],
+		[
+			"levels.vaccines[*].type",
+			"CurrentStock.type"
+		],
+		[
+			"levels.vaccines[*].count",
+			"CurrentStock.count"
+		],
+		[
+			"report_datetime",
+			"CurrentStock.date"
+		]
+	]
+}
```

Comment:
I think that `tabs` love you :stuck_out_tongue:

Reply:
You know what, I think it's because sometimes I use jsonlint...

Reply:
You can indicate the `--indent` option :see_no_evil: 

https://github.com/zaach/jsonlint/